gqt/cgrouper/cgrouper.go

```diff
@@ -17,7 +19,7 @@ func GetCGroupPath(cgroupsRootPath, subsystem, tag string, privileged, throttlin
 	}
 
 	if throttlingCPU {
-		parentCgroup = filepath.Join(parentCgroup, cgroups.GoodCgroupName)
+		parentCgroup = filepath.Join(parentCgroup, gardencgroups.GoodCgroupName)
 	}
```

Comment:
Is there no Throttling in cgroups v2?

Reply:
cgroup v2 has CPU throttling as well. For both v1 and v2 we are doing this ourselves, by moving processes between bad and good cgroups (see throttler.go)